recipe/build.sh

```diff
@@ -1,18 +1,15 @@
 #!/bin/bash
 
-if [[ $(uname) == Linux ]]; then
-  # FIXME: This is a terrible workaround.
-  # Ideally we should fix the information the .la files.
-  rm -rf $PREFIX/lib/libquadmath.la
-  rm -rf $PREFIX/lib/libgfortran.la
```

Comment:
The default channel had some `.la` files with hard-coded paths. they probably fixed that. Since the build is passing without them now let's remove.